src/ocamlorg_package/lib/info.ml

```diff
@@ -137,9 +138,22 @@ let mk_revdeps pkg pkgs rdepends =
     []
   |> Lwt.map List.rev
 
+module String_map = Map.Make (String)
+
+let get_publication package =
+  let name = OpamPackage.name_to_string package in
+  let name_with_version = OpamPackage.to_string package in
+  let path = Printf.sprintf "packages/%s/%s/opam" name name_with_version in
```

Comment:
`name_with_version` is itself a key - it'd be simpler if map were set-up so that `name.version` was already the key rather than having to put `packages/name/name.version/opam` back in